index.jsx

```diff
@@ -58,20 +58,10 @@ export default {
         },
     },
     onInit: dispatch => {
-        document.ondrop = event => {
-            event.preventDefault();
-        };
-
-        document.ondragover = document.ondrop;
-
         document.body.ondragover = event => {
-            const dragOverEvent = event;
-            if (!dragOverEvent.dataTransfer.files.length) {
-                dragOverEvent.dataTransfer.dropEffect = 'none';
-                dragOverEvent.dataTransfer.effectAllowed = 'none';
-            } else {
-                dragOverEvent.dataTransfer.effectAllowed = 'uninitialized';
-            }
+            const ev = event;
```

Comment:
Is this ev assignment needed? It still references the same object.

Reply:
Yes, otherwise it would be parameter assignment which is not allowed.